inductiva/simulators/README.md

```diff
@@ -0,0 +1,235 @@
+# Simulators
+
+Inductiva API has available several open-source simulators ready to use. Users that are familiar with the simulators can easily start running simulations with their previously prepared simulation configuration files. In this way, they can take advantage of performant hardware to speed up their simulation and exploration.
+No installation or management of the simulators is required, no need to worry about the hardware, and no need to worry about the software dependencies. Inductiva API takes care of all of that for you.
```

Comment:
No installation or management of the simulators is required, no need to worry about  hardware or software dependencies. Inductiva API takes care of all of that for you.